controller.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/spf13/cobra"
+
+	orgv1 "github.com/appuio/control-api/apis/organization/v1"
+	controlv1 "github.com/appuio/control-api/apis/v1"
+
+	"github.com/appuio/control-api/controllers"
+	//+kubebuilder:scaffold:imports
+)
+
+// ControllerCommand creates a new command allowing to start the controller
+func ControllerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "controller",
+	}
+
+	zapfs := flag.NewFlagSet("zap", flag.ExitOnError)
+	opts := zap.Options{}
+	opts.BindFlags(flag.CommandLine)
+	cmd.Flags().AddGoFlagSet(zapfs)
+
+	metricsAddr := cmd.Flags().String("metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	enableLeaderElection := cmd.Flags().Bool("leader-elect", false,
```

Comment:
Why don't we enable leader election by default?

Reply:
It's the default from kubebuilder. I don't see a need for the controller to be HA but I also would not mind enabling it by default.